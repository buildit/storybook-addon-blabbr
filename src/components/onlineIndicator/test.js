import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import OnlineIndicator from './';

test('Online indicator renders correctly when system is offline', () => {
  const tree = renderer.create(<OnlineIndicator isOnline={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Online indicator renders correctly when system is online', () => {
  const tree = renderer.create(<OnlineIndicator isOnline />).toJSON();
  expect(tree).toMatchSnapshot();
});
