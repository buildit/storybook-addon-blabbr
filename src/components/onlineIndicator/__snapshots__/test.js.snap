// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Online indicator renders correctly when system is offline 1`] = `
<div
  className="blabbr-status-indicator offline"
  data-for="blabbr-indicator-tooltip"
  data-tip={true}
>
  <div
    className="__react_component_tooltip place-top type-dark "
    data-id="tooltip"
  >
    
  </div>
</div>
`;

exports[`Online indicator renders correctly when system is online 1`] = `
<div
  className="blabbr-status-indicator online"
  data-for="blabbr-indicator-tooltip"
  data-tip={true}
>
  <div
    className="__react_component_tooltip place-top type-dark "
    data-id="tooltip"
  >
    
  </div>
</div>
`;
